### ISSUE TITLE
Animate point along a route example incorrectly links to turf

### ISSUE THREAD

**Issue by samfader** [mapbox/mapbox-gl-js#6206]
**mapbox-gl-js version**: 0.44.1

### Steps to Trigger Behavior

See the [Animate point along a route](https://www.mapbox.com/mapbox-gl-js/example/animate-point-along-route/) GL JS example. On line 39:

`<script src='//api.tiles.mapbox.com/mapbox.js/plugins/turf/v2.0.0/turf.min.js' charset='utf-8'></script>`

### Expected Behavior

The correct URL is:

`<script src='https://api.tiles.mapbox.com/mapbox.js/plugins/turf/v3.0.11/turf.min.js'></script>`

(I'm not positive if the version specifically needs to be 2.0.0 for this example, but it looks like we use 3.0.11 in [this](https://www.mapbox.com/mapbox-gl-js/example/mapbox-gl-draw/) example).

In short, it looks like the `https:` was missed and possibly the version needs to be updated.

### Actual Behavior

Maps using this example will not load due to the malformed URL.

cc @colleenmcginnis

**Comment by jfirebaugh** [comment]
2bdee67a44d69e8a45fd20ab1ac5d117cae32946
